gto-support-db/src/main/java/org/ccci/gto/android/common/db/AbstractDao.java

```diff
@@ -236,10 +239,9 @@ public final Cursor getCursor(@NonNull final Class<?> clazz, @NonNull final Stri
                 projection[i] = projection[i].contains(".") ? projection[i] : prefix + projection[i];
             }
 
-            // prefix an un-prefixed orderBy field
-            if (orderBy != null && !orderBy.contains(".")) {
-                orderBy = prefix + orderBy;
-            }
+            // prefix un-prefixed clauses
+            addPrefixToFields(orderBy, prefix);
```

Comment:
you lost the test to see if orderBy already has a prefix, this will end up breaking the measurements app, I know I utilize an order by with a prefix. This also lost the test for null orderBy.